source/draw/gpu/GpuYuv420Semiplanar.ooc

```diff
@@ -16,23 +16,29 @@
 
 use ooc-math
 use ooc-draw
-import GpuMonochrome, GpuCanvas, GpuUv, GpuContext, GpuImage
+import GpuContext, GpuImage, GpuSurface, GpuCanvas
 
 GpuYuv420Semiplanar: class extends GpuImage {
-	_y: GpuMonochrome
+	_y: GpuImage
 	y ::= this _y
-	_uv: GpuUv
+	_uv: GpuImage
 	uv ::= this _uv
-
+	filter: Bool {
+		get
```

Comment:
Okay, good. For a moment there, I thought rock figured it out by itself and it seemed a little crazy.